src/components/ServiceQuestions.js

```diff
@@ -0,0 +1,245 @@
+        if (this.state.pageNumber !== 1) {
+            var temp = this.state.pageNumber - 1;
+            this.setState({pageNumber: temp })
+        }
+    }
+
+    nextPage() {
+        var totalPages = Math.ceil(this.state.serviceQuestions.length / this.state.itemsPerPage);
+        if (this.state.pageNumber + 1 <= totalPages) {
+            var temp = this.state.pageNumber + 1;
+            this.setState({pageNumber: temp })
+        }
+    }
+
+    choosePage(newPage) {
+        this.setState({pageNumber: newPage })
+
+    }
+
+    createPageButtons() {
+        var totalPages = Math.ceil(this.state.serviceQuestions.length / this.state.itemsPerPage);
+        var i;
+
+        for (i = 1; i < totalPages + 1; i++) {
+            if ((document.getElementById("Button " + i)) == null)
+            {
+                console.log(i);
+                var button = document.createElement("BUTTON");
+                var buttonText = document.createTextNode("" + i);
+                button.appendChild(buttonText);
+                button.type = "button";
+                button.id = "Button " + i;
+                button.value = i;
+                button.onclick = function(e) {
+                    this.setState({pageNumber: e.target.value})
+                }.bind(this);
+                document.getElementById("pageButtons").appendChild(button);
+            }
+        }
+    }
+
+    addServiceQuestion() {
+        console.log('Add Service Click happened');
+        const titleInput = document.getElementById('titleInput');
+        const questionInput = document.getElementById('questionInput');
+        var title = titleInput.value;
+        var question = questionInput.value;
+        if (title === "" || question === "") {
+            alert("Please add a question")
+        }
+        this.serviceQuestionsService
+            .addServiceQuestion({title: title, question: question})
+            .then(() => this.refreshServiceQuestions());
+        titleInput.value = "";
+        questionInput.value = "";
+    }
+
+    updateServiceQuestion(id) {
+        const titleInput = document.getElementById('titleInput');
+        const questionInput = document.getElementById('questionInput');
+        var title = titleInput.value;
+        var question = questionInput.value;
+        this.serviceQuestionsService
+            .updateServiceQuestion({title: title, question: question}, id)
+            .then(() => this.refreshServiceQuestions());
+        titleInput.value = "";
+        questionInput.value = "";
+    }
+
+    deleteServiceQuestion(id) {
+        console.log('Delete ServiceQuestion Click happened');
+        this.serviceQuestionsService.deleteServiceQuestion(id)
+            .then(() => this.refreshServiceQuestions());
+    }
+
+    render() {
+        let div = (
+            <div>
+                <h3>Service Questions</h3>
+                <table className="table">
+                    <tr>
+                        <th>Title</th>

```

Comment:
Throughout the columns are still named based on the columns for the FAQ table not the ServiceQuestion table. This happens further down with `serviceQuestion.title` too. You just gotta rename the column headers and what column you're pulling from.